Neos.ContentRepository.Core/Classes/Feature/Common/ConstraintChecks.php

```diff
@@ -602,63 +584,27 @@ protected function requireNodeAggregateToBeChild(
         );
     }
 
-    /**
-     * @throws NodeNameIsAlreadyOccupied
-     */
-    protected function requireNodeNameToBeUnoccupied(
```

Comment:
a i see thats what you meant, we exchange this check for the simpler, non dimension aware `requireNodeNameToBeUncovered` thus this method can be removed and its callers rewritten

Reply:
exactly, this was only needed for checking that if two node aggregates with the same name share a parent, they must not share dimension space points. Which is no longer allowed in the first place.

Reply:
Okay i see but my feeling says `requireNodeTypeNotToDeclareTetheredChildNodeName` is not called often enough? Only once in the Move handling?